templates/web/fixmystreet-uk-councils/about/privacy.html

```diff
@@ -476,4 +476,6 @@ <h2 id="cookies">Cookies</h2>
     </tr>
 </table>
 
+[% TRY %][% INCLUDE 'about/googlecookies.html' %][% CATCH file %][% END %]
```

Comment:
Don't think you need the `[% END %]` at the end of this line.

Reply:
It is required, it closes the `TRY`.

Reply:
Ah so it is, my bad!